pkg/start/start.go

```diff
@@ -153,16 +178,97 @@ func (o *Options) Run() error {
 	return nil
 }
 
+func (o *Options) handler() http.Handler {
+	mux := http.NewServeMux()
+	mux.Handle("/metrics", promhttp.Handler())
+	return mux
+}
+
+func (o *Options) makeTLSConfig() (*tls.Config, error) {
+	if !fileExists(o.ServingCertFile) {
```

Comment:
Do we need these guards?  I'd expect the TLS consumer would error out with a reasonable message if the cert or key file did not exist.

Reply:
Actually, right after these guards you load the bytes, and `ReadFile` returns great error messages [like][1]:

```
open /does/not/exist: no such file or directory
```

So we can definitely drop these guards.

[1]: https://play.golang.org/p/Yc0PxnjRvmH